### PR TITLE
apps/* cleanup

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

**Comment by robhudson** [comment]
Awesome work. I can give this a closer look on Monday.

**Comment by robhudson** [comment]
I looked through each commit and couldn't find anything to complain about. :grinning:  Consider that an r+ from me. Merge when you're ready.

And excellent work. As someone new to the repo this consolidation helps a lot.

Also, I may want to move the recently added `absolutify` to the `kuma.core` app. Are there other things that should be in core that aren't currently that could be moved later?

**Comment by jezdez** [comment]
@robhudson Awesome, thank you for the code review!

I'm sure there are plenty of utilities in the individual apps, mostly search and wiki that can be centralized in kuma.core since they are general purpose anyway. I'm not sure yet of the kuma.landing app should stay the way it is right now, since it contains a bunch of unrelated utils. In either way this should be done in a separate ticket/PR.

**Comment by lmorchard** [comment]
Well done!

**Comment by jezdez** [comment]
@lmorchard :+1:

**Comment by groovecoder** [comment]
@lmorchard :+1:

File: kuma/actioncounters/migrations/0003_update_unique_hashes.py [hunk 1]
```diff
@@ -16,8 +16,8 @@ def forwards(self, orm):
         # but wasn't implemented before this migration. Doing it now, so the
         # data migration has a smaller set to work with.
         (orm.ActionCounterUnique.objects
-                .filter(user=None, modified__lt=datetime.datetime(2011,8,1))
-                .delete())
+            .filter(user=None, modified__lt=datetime.datetime(2011, 8, 1))
+            .delete())
 
         # Update all remaining counters with a unique hash.
         from actioncounters.utils import get_unique
```
Comment (groovecoder) [inline kuma/actioncounters/migrations/0003_update_unique_hashes.py]: @jezdez I had to change this line to `from kuma.actioncounters.utils import get_unique` to make migrations run during vagrant provision. Is that a safe change to make permanent in `master` branch?

Comment (jezdez) [inline kuma/actioncounters/migrations/0003_update_unique_hashes.py]: Yes.

**Pull request merged**